File: src/main/java/fr/tikione/c2e/model/web/Picture.java
package fr.tikione.c2e.model.web;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Picture url and its optional legend.
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Picture {
    
    private String url;
    private String legend;
}

File: src/main/kotlin/fr/tikione/c2e/model/web/Picture.kt
package fr.tikione.c2e.model.web

/**
 * Picture url and its optional legend.
 */
class Picture {

    var url: String? = null
    var legend: String? = null

    constructor() {
    }

    constructor(url: String?, legend: String?) {
        this.url = url
        this.legend = legend
    }

    override fun toString(): String {
        return "Picture(url=$url, legend=$legend)"
    }
}
